moveit_planners/pilz_industrial_motion_planner/src/trajectory_generator_lin.cpp

```diff
@@ -52,7 +52,7 @@
 namespace pilz_industrial_motion_planner
 {
 TrajectoryGeneratorLIN::TrajectoryGeneratorLIN(const moveit::core::RobotModelConstPtr& robot_model,
-                                               const LimitsContainer& planner_limits)
+                                               const LimitsContainer& planner_limits, const std::string& /*group_name*/)
```

Comment:
Is it too much of a hassle to store the group from the constructor for `LIN` and `CIRC` as well?
It's very inconsistent to ignore the parameter in two out of three constructors.

Reply:
The group is only needed in `TrajectoryGeneratorPTP` to find joint limits for that group only.
As a consequence of [Pilz' software design](https://github.com/ros-planning/moveit/pull/2940/commits/c90aacbeb045292e01855938e059c72e194dafc8#diff-7b18543ba76a984ef64270ec9893a6f52c042a9cd3c9a233de62e2f49ccb1de5R66), I had to introduce the parameter in all other constructors as well :disappointed: 

The group itself isn't stored in any of those classes.